.evergreen.yml

```diff
@@ -346,3 +394,25 @@ buildvariants:
       - name: build_e2e_image
       - name: build_testrunner_image
       - name: build_prehook_image
+
+  - name: release_blocker
+    display_name: release_blocker
+    run_on:
+      - ubuntu1604-packer  # Note: cheapest machine I found
```

Comment:
[q] does this one have docker installed the same way? I would lean in favour of just using the same machines that we use for our tests just to keep the variety of environments as low as possible.

Reply:
This runs only on the release blocker. The task that releases the images uses `ubuntu1604-test` ass all the other tasks. Here we just need a machine which can ran the automatically failing task